app/Http/Controllers/MonitorController.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use DateTime;
+use DateInterval;
+use DatePeriod;
+use App\Enums\ClassicPalette;
+use App\Enums\HighContrastPalette;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use stdClass;
```

Comment:
`stdClass` is included elsewhere (such as in the `DB` class), so it isn't necessary here